Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -0,0 +1,72 @@
+---
+versionFrom: 8.0.0
+---
+
+## Non-synchronised/replicated file system
```

Comment:
I might be misreading this but is this

Non-synchronised/non-replicated file system?

I'm not sure what is meant by the slash

particulary with the next heading being

syncronised/replicated or shared file system

Reply:
After many late nights debate, we have settled on "Standalone File System"